File: angular/src/app/models/physician.model.ts
import { Address } from './address.model';
import { UserAccount } from './user-account.model';

export class Physician {
    public firstName: string;
    public lastName: string;
    public phoneNumber: string;
    public address: Address;
    public userAccount: UserAccount;
    public createdOn: Date;
    public modifiedOn: Date;
}

File: dotnet/PhysiciansReach/PR.Models/PhysicianModel.cs
using System;

namespace PR.Models
{
    public class PhysicianModel
    {
        public UserAccountModel UserAccount { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string PhoneNumber { get; set; }

        public string NPI { get; set; }

        public string DEA { get; set; }

        public AddressModel Address { get; set; }

        public DateTime CreatedOn { get; set; }

        public DateTime ModifiedOn { get; set; }
    }
}
